### ISSUE TITLE
About the batchsize

### ISSUE THREAD

**Issue by meisenwang** [Zizzzzzzz/SMamba_AAAI2025#4]
> On my server, both SMamba and RVT utilized 2 NVIDIA 3090 GPUs for training with a batch size of 6. The training speed of SMamba was 1.75 s/it, while the training speed of RVT was 1.5 it/s. 

 _Originally posted by @Zizzzzzzz in [#2](https://github.com/Zizzzzzzz/SMamba_AAAI2025/issues/2#issuecomment-2677832015)_
Great works! Congratulations!
The batch size in your github is 8 per gpu, but two 3090s cannot run on the gen1 dataset. The batch size you mentioned in the issue is 6. I want to know whether the total batch size is 6 or the batch size for each gpu is 6? Is the batch size of 6 a parameter that can reproduce the thesis point?
Thank you for your reply!

**Comment by Zizzzzzzz** [comment]
Thanks. The batch size for each gpu is 6. You can adjust the batch size according to your device, and then modify the learning rate accordingly using this formula:
lr = 2e-4 * sqrt(effective_batch_size/8).

**Comment by meisenwang** [comment]
Thank you very much!
